jobs/ci-pipeline-defaults.yml

```diff
@@ -34,8 +34,8 @@
           branches:
               - master
       - git:
-          url: "https://github.com/CentOS-PaaS-SIG/aos-ci"
-          basedir: aos-ci
+          url: "https://github.com/CentOS-PaaS-SIG/{repo-name}"
+          basedir: '{repo-name}'
```

Comment:
So do all the jobs need a repo-name: defined in them now?  I see it in rpmbuild_trigger.yaml, but not the others.  Not sure if I'm missing something.. haven't run this through jenkins-jobs test so not sure if it resolves